.ci/magician/cmd/test_terraform_vcr_record_replay.tmpl

```diff
@@ -0,0 +1,31 @@
+{{- if gt (len .RecordingResult.PassedTests) 0 -}}
+$\textcolor{green}{\textsf{Tests passed during RECORDING mode:}}$
+{{range .RecordingResult.PassedTests}}`{{.}}`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-{{$.PRNumber}}/artifacts/{{$.BuildID}}/recording/{{.}}.log)]
+{{end}}
+
+{{- if gt (len .ReplayingAfterRecordingResult.FailedTests ) 0 -}}
+$\textcolor{red}{\textsf{Tests failed when rerunning REPLAYING mode:}}$
+{{range .ReplayingAfterRecordingResult.FailedTests}}`{{.}}`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-{{$.PRNumber}}/artifacts/{{$.BuildID}}/build-log/replaying_build_after_recording/{{.}}_replaying_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-{{$.PRNumber}}/artifacts/{{$.BuildID}}/replaying_after_recording/{{.}}.log)]
+{{end}}
+
+Tests failed due to non-determinism or randomness when the VCR replayed the response after the HTTP request was made.
+
+Please fix these to complete your PR. If you believe these test failures to be incorrect or unrelated to your change, or if you have any questions, please raise the concern with your reviewer.
+
+{{else}}
+$\textcolor{green}{\textsf{No issues found for passed tests after REPLAYING rerun.}}$
+{{end}}{{/* end of if gt (len .ReplayingAfterRecordingResult.FailedTests ) 0 */}}
+---
```

Comment:
Looks like we'll need a new line above `---`, otherwise, the font size of the previous line gets weird. see https://github.com/GoogleCloudPlatform/magic-modules/pull/10910#issuecomment-2155353048

Reply:
It was fixed already. The first few tests had strange font size, so I fixed it, check the latest comment, they are similar messages, but the format is fixed in the latest one.

Reply:
oh, right. Sorry about it. I somehow read that as the latest version. Thanks for the quick response!